proto/evmos/auctions/v1/auction.proto

```diff
@@ -0,0 +1,23 @@
+// Copyright Tharsis Labs Ltd.(Evmos)
+// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
+syntax = "proto3";
+package evmos.auctions.v1;
+import "gogoproto/gogo.proto";
+
+option go_package = "github.com/evmos/evmos/v18/x/auctions/types";
+
+// Bid is a message type for submitting a bid
+message Bid {
+  // sender is the address of the bidder
+  string sender = 1 [(gogoproto.jsontag) = "sender", (gogoproto.moretags) = "yaml:\"sender\""];
+  // amount is the amount of the bid
+  string amount = 2 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Coin", (gogoproto.nullable) = false];
+}
+
+// Deposit is a message type for deposits
+message Deposit {
+  // sender is the address of the depositor
+  string sender = 1 [(gogoproto.jsontag) = "sender", (gogoproto.moretags) = "yaml:\"sender\""];
+  // amount is the amount of the deposit
+  string amount = 2 [(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Coin", (gogoproto.nullable) = false];
```

Comment:
nit: I would also rename amount to something like bid_value to avoid this:

```go
if bid.Amount.Amount.LTE(lastBid.Amount.Amount) {
```

amount is already a field of the coin.

Reply:
reopening this one because no answer here. In Cosmos SDK Amount is referred to only the Int

Reply:
It is better to change it now instead of in the future because it will require a migration